extensions/amp-list/0.1/amp-list.js

```diff
@@ -181,17 +183,48 @@ export class AmpList extends AMP.BaseElement {
     });
 
     if (this.loadMoreEnabled_) {
-      this.getLoadMoreButton_();
-      this.getLoadMoreLoadingElement_();
-      this.getLoadMoreFailedElement_();
-      this.getLoadMoreEndElement_();
-      this.getLoadMoreButtonClickable_();
-      this.getLoadMoreFailedClickable_();
-      // Hide overflow element
-      const overflowElement = this.getOverflowElement();
-      if (overflowElement) {
-        toggle(overflowElement, false);
-      }
+      this.mutateElement(() => {
+        this.initializeLoadMoreButton_();
+        this.initializeLoadMoreLoadingElement_();
+        this.initializeLoadMoreFailedElement_();
+        // Hide overflow element
+        const overflowElement = this.getOverflowElement();
+        if (overflowElement) {
+          toggle(overflowElement, false);
+        }
+      }).then(() => {
+        this.adjustContainerForLoadMoreButton_();
+      });
+    }
+  }
+
+  /**
+   * @private
+   * @return {!Element}
+   */
+  getEndOfList_() {
+    if (!this.endOfListDummy_) {
```

Comment:
I assume this was added because load-more was changed to be position absolute and is now at the bottom of the list, per comment above, if we stop having container fill the list, it would be correct and you can keep monitoring the load more.

Reply:
Done.